File: example/src/main/java/com/thesurix/example/gesturerecycler/adapter/HeaderViewHolder.java
package com.thesurix.example.gesturerecycler.adapter;

import com.thesurix.example.gesturerecycler.R;
import com.thesurix.gesturerecycler.GestureViewHolder;

import android.view.View;
import android.widget.TextView;

import butterknife.BindView;
import butterknife.ButterKnife;


public class HeaderViewHolder extends GestureViewHolder {

    @BindView(R.id.header_text)
    TextView mHeaderText;

    public HeaderViewHolder(final View view) {
        super(view);
        ButterKnife.bind(this, view);
    }

    @Override
    public boolean canDrag() {
        return false;
    }

    @Override
    public boolean canSwipe() {
        return false;
    }
}

File: example/src/main/java/com/thesurix/example/gesturerecycler/adapter/HeaderViewHolder.kt
package com.thesurix.example.gesturerecycler.adapter

import android.widget.TextView
import com.thesurix.example.gesturerecycler.databinding.HeaderItemBinding
import com.thesurix.example.gesturerecycler.model.MonthHeader
import com.thesurix.gesturerecycler.GestureViewHolder


class HeaderViewHolder(binding: HeaderItemBinding) : GestureViewHolder<MonthHeader>(binding.root) {

    val headerText: TextView = binding.headerText

    override fun canDrag() = false

    override fun canSwipe() = false

    override fun bind(item: MonthHeader) {
        headerText.text = item.name
    }
}
